context/context.go

```diff
@@ -19,9 +19,16 @@ import (
 var sctplbContext = SctplbContext{}
 
 type SctplbContext struct {
-	RanPool sync.Map // map[net.Conn]*Ran
+	RanPool  sync.Map // map[net.Conn]*Ran
+	Backends []interface{}
```

Comment:
Moving the Backend list into sctplb's context can extend the flexibility of the source code (for example: implement different scheduling algorithms).